raiden_contracts/tests/test_channel_unlock.py

```diff
@@ -326,7 +652,76 @@ def test_channel_unlock(
     pre_balance_B = custom_token.functions.balanceOf(B).call()
     pre_balance_contract = custom_token.functions.balanceOf(token_network.address).call()
 
-    # TODO to be moved to a separate test
+    # Unlock the tokens
+    token_network.functions.unlock(
+        A,
+        B,
+        pending_transfers_tree.packed_transfers,
+    ).transact()
+
+    balance_A = custom_token.functions.balanceOf(A).call()
+    balance_B = custom_token.functions.balanceOf(B).call()
+    balance_contract = custom_token.functions.balanceOf(token_network.address).call()
+    assert balance_A == pre_balance_A + 9
+    assert balance_B == pre_balance_B + 6
+    assert balance_contract == pre_balance_contract - values_B.locked
+
+
+def test_unlock_channel_event(
```

Comment:
Consider placing the event test last in the file. I used this convention for each feature test file.

Reply:
It is last in the file